include/ck_tile/ops/fmha/block/block_dropout.hpp

```diff
@@ -22,20 +22,27 @@ struct NullBlockDropout
     }
 };
 
+template <bool IsDropout_ = true, bool IsWG32_ = true, bool IsStoreRandval_ = false>
 struct BlockDropout
 {
+    static constexpr bool IsDropout = IsDropout_;
+    // true:  32*32 warp gemm
+    // false: 16*16 warp gemm
+    static constexpr bool IsWG32         = IsWG32_;
+    static constexpr bool IsStoreRandval = IsStoreRandval_;
+
```

Comment:
Suggest add 
```C++
static constexpr bool IsDropout = false;
static constexpr bool IsStoreRandval = false; 
```
to `NullBlockDropout` and use `NullBlockDropout` when  `kHasDropout == false`, because using `FmhaDropout` has some constructing cost

Reply:
Refactored.